.github/workflows/run-tests.yml

```diff
@@ -10,6 +10,23 @@ jobs:
             - uses: actions/setup-node@v2
               with:
                   node-version-file: ".nvmrc"
+            - uses: actions/cache@v2
```

Comment:
This should shave up to a minute while running tests when the dependencies don't change.